### PR TITLE
Minor corrections in Fw nav.

### PR DESCRIPTION
**Pull request by EmilsPa**
Minor corrections in Fw nav.
Removed some blankspaces.

### CODE REVIEW

File: src/fw_nav.c [hunk 1]
```diff
@@ -200,11 +200,12 @@ void fw_nav(void)
 
         // Elevator compensation depending on behaviour.
         // Prevent stall with Disarmed motor
-        if (!f.CLIMBOUT_FW)
-            GPS_angle[PITCH] -= (abs(angle[ROLL]) *cfg.fw_roll_comp);
+        if(f.MOTORS_STOPPED)
```
Comment (trollcop) [inline src/fw_nav.c]: spacing

File: src/fw_nav.c [hunk 2]
```diff
@@ -200,11 +200,12 @@ void fw_nav(void)
...
+            GPS_angle[PITCH]=constrain(GPS_angle[PITCH],0, cfg.fw_gps_maxdive*10);
 
         // Add elevator compared with rollAngle
-        GPS_angle[PITCH] -= abs(angle[ROLL]);
+        if (!f.CLIMBOUT_FW)
+            GPS_angle[PITCH] -= (abs(angle[ROLL]) *cfg.fw_roll_comp);
```
Comment (trollcop) [inline src/fw_nav.c]: spacing

**Comment by trollcop** [comment]
ok  to test

**Comment by trollcop** [comment]
Please make sure when you commit stuff the spacing/indentation follows code guidelines.
it's very very important.

**Comment by EmilsPa** [comment]
Ok sorry.
Fixed now.

**Comment by trollcop** [comment]
There are still issues, read all the line notes.
if() needs space, there's space between key=val, etc.

**Comment by EmilsPa** [comment]
Can't imagine how many errors it's possible to make on four lines... :(

**Comment by trollcop** [comment]
I commented all of them, it's not that hard.

did you read code style guidelines in the wiki? sigh.

**Comment by trollcop** [comment]
@fiendie can you educate this guy? ;)

**Comment by readerror67** [comment]
Hey, I will take a look at this tonight at work and try to help Patrick get
things sorted.

**Comment by trollcop** [comment]
btw, there are still formatting issues I just dont have time to deal with it.
before opening new PR *PLEASE* make sure the code is properly formatted.

**Pull request merged**